src/Runner.Worker/ActionManifestManager.cs

```diff
@@ -454,9 +454,9 @@ public ActionDefinitionData Load(IExecutionContext executionContext, string mani
                     case "steps":
                         if (!string.IsNullOrEmpty(Environment.GetEnvironmentVariable("TESTING_COMPOSITE_ACTIONS_ALPHA")))
                         {
-                            var steps = run.Value.AssertSequence("steps");
-                            var evaluator = executionContext.ToPipelineTemplateEvaluator();
-                            stepsLoaded = evaluator.LoadCompositeSteps(steps);
+                            var stepsToken = run.Value.AssertSequence("steps");
+                            steps = PipelineTemplateConverter.ConvertToSteps(templateContext, stepsToken);
```

Comment:
I changes this to use PipelineTemplateConverter instead of PipelineTemplateEvaluator